app/com/linkedin/drelephant/ElephantRunner.java

```diff
@@ -179,7 +176,10 @@ public void run() {
         logger.info(String.format("Analyzing %s", analysisName));
         AppResult result = _analyticJob.getAnalysis();
         result.save();
-        logger.info(String.format("Analysis of %s took %sms", analysisName, System.currentTimeMillis() - analysisStartTimeMillis));
+        long processingTime = System.currentTimeMillis() - analysisStartTimeMillis;
+        logger.info(String.format("Analysis of %s took %sms", analysisName, processingTime));
+        MetricsController.setJobProcessingTime(processingTime);
```

Comment:
Shoot ok I went to implement this and realized it doesn't really make sense. At this level of abstraction (i.e. ElephantRunner dealing with AppResults) there's no knowledge of tasks.